PSOperationsTests/PSOperationsTests.swift

```diff
@@ -973,4 +1006,40 @@ class PSOperationsTests: XCTestCase {
         XCTAssertEqual(1, opQ.operationCount)
         XCTAssertTrue(op.waitCalled)
     }
+    
+    func testOperationsNotExecuting() {
+        
+        var opCount = 0
```

Comment:
I see. So if you reach 25,000 then nothing froze and you're good. Perhaps a comment about the freezing down by the timeout would help a noob like me.